### DISCUSSION TITLE
BlueSCSI 1.0-b to 1.0-c

### DISCUSSION THREAD

**Discussion by keivanshahin** [erichelgeson/BlueSCSI-v1#26] [Q&A] [answered]
Instead of cuting cant i just skip adding dioed to D5 and then just link TERM to 5v BluePill after?

I know it says "Make sure continiutiy from TERMPOWER to the diode still flows correctly but is disconnected from the jumper." but does it really do anything as one leg is only connected.

**Comment by erichelgeson** [comment, marked as answer]
You cut the connection prior to the diode - then connect it directly to the 5v from the bluepill in the next step - so both legs of the jumper are connected.

**Reply by keivanshahin** [reply]
sorry my eyes didnt work today :) now i se the connection why it need to be cut ;)

**Reply by erichelgeson** [reply]
No worries! Happy bodging :)
